src/main/java/com/inventor/email/EmailControl.java

```diff
@@ -0,0 +1,49 @@
+package com.inventor.email;
+
+import com.inventor.email.settings.EmailSettings;
+import com.inventor.email.service.ServicesE;
+import lombok.AllArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Date;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
```

Comment:
I believe it should be `/api/emails` in order for this path to be propagated to all methods defined in this controller